components/automate-ui/src/app/pages/team-add-users/team-add-users.component.html

```diff
@@ -0,0 +1,50 @@
+<chef-page
+  subheading="Automate only displays local users."
+  [attr.secondary-btn-show]="showSecondary"
+  [attr.heading]="getHeading()"
+  [attr.confirm-btn-text]="getConfirmBtnText()"
+  [attr.page-loading]="loading"
+  [attr.confirm-loading]="addingUsers"
+  [attr.disable-confirm]="usersToAddValues().length === 0"
+  [attr.error-message]="getErrorMessage()"
+  (confirm)="addUsers()"
+  (close)="closePage()">
+  <div *ngIf="!loading">
+    <div id="no-users-container" *ngIf="usersNotFiltered()?.length === 0">
```

Comment:
Could `usersNotFiltered` ever return not-a-list? I don't think so, so the question mark should not be needed. Same on L16.